src/main/java/com/ldtteam/structures/helpers/Settings.java

```diff
@@ -309,7 +309,7 @@ public void setActiveSchematic(final Blueprint blueprint)
     {
         if (blueprint == null)
         {
-            reset();
+            resetBlueprint();
```

Comment:
reason?

Reply:
The `blueprint` field needs to be cleared otherwise the preview remains visible after closing the GUI.

However, I didn't want to clear the other fields, so that you can quickly place multiple copies of the same shape.  (It does still lose the mirror/rotation somewhere or other, but all other properties are preserved.)

Reply:
Yeah, but this method is called from different places and you weakened its effect without making it up elsewhere

Reply:
It's called from exactly one other place, where it is considered an error for blueprint to be null (though not impossible).  It seemed relatively harmless.  And it seems incorrect to clear all the other properties from a blueprint-setter method.

Reply:
Well I would rather see a softreset method for shapetool but ok